AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt

```diff
@@ -3911,6 +3911,10 @@ lg.com#@#.cookie-check
 mycard2go.com#@#.cookiebar
 foodnetwork.co.uk#@#.cookie-message:not(html):not(body)
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/177450
+!+ PLATFORM(ios, ext_android_cb, ext_safari)
+@@||shop.blu-blumen.de/Scripts/CookieBanner.js
+!
```

Comment:
Please move it to the top, somewhere here - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt#L17-L20

Reply:
done